src/api/Alert/types.ts

```diff
@@ -102,6 +102,7 @@ export interface IAlertQuery extends AlertQueryData {
   query: string;
   dialect: GraphQueryDialect;
   updatedAt: Date;
+  alertId: number;
```

Comment:
why adding this? the query is normally already inside the Alert type, so the alertId should be accessible

Reply:
Do we need to modify the api response for this reason ? Since the alert object contains alert queries.
I see that you haven't modified the API doc to reflect that.